### PR TITLE
Fixes for when multistage isn't being used

### PR DESCRIPTION
**Pull request by aub**
The current code assumes the usage of capistrano multistage. In particular, the #stage method is used, which crashes in its absence. I've changed this to ignore stage-related stuff when calling that method results in a NameError.

### CODE REVIEW

File: lib/capistrano/recipes/deploy_lock.rb [hunk 1]
```diff
@@ -121,8 +121,15 @@ def remove_deploy_lock
         next
       end
 
+      stg = nil
+      begin
+        stg = stage
```
Comment (ndbroadbent) [inline lib/capistrano/recipes/deploy_lock.rb]: Could you please use `respond_to?(:stage)` instead?

File: lib/capistrano/recipes/deploy_lock.rb [hunk 2]
```diff
@@ -121,8 +121,15 @@ def remove_deploy_lock
...
+      rescue NameError
+        # this means we don't have multistage. ok.
+      end
+
       # Unexpired lock is present, so display the lock message
-      logger.important Capistrano::DeployLock.message(application, stage, deploy_lock)
+      logger.important Capistrano::DeployLock.message(application, stg, deploy_lock)
```
Comment (ndbroadbent) [inline lib/capistrano/recipes/deploy_lock.rb]: Maybe something like:

```
 logger.important Capistrano::DeployLock.message(application, (respond_to?(:stage) ? stage : nil), deploy_lock)
```

File: lib/capistrano/recipes/deploy_lock.rb (hunk 1 above)
Comment (aub) [inline lib/capistrano/recipes/deploy_lock.rb]: Sure, I pushed your suggested change. Agreed that it's nicer this way.

**Comment by ndbroadbent** [comment]
Great, thanks!

**Comment by ndbroadbent** [comment]
Released in 0.3.1

**Pull request merged**